### PR TITLE
Search.GenerateAuxiliaryData should copy Downloads.v1.json to search metadata location 

### PR DESCRIPTION
**Pull request by skofman1**
Issue: https://github.com/nuget/engineering/issues/1320

Search.GenerateAuxiliaryData will copy Downloads.v1.json  from statistics reports location to search service metadata location.
Currently this logic is in Stats.CreateAzureCdnWarehouseReports, which stores the report in gallery blob storage and in another location (search metadata container). With this change every geo-region search is deployed in will copy Downloads.v1.json to it's blob storage.

### CODE REVIEW

File: src/Search.GenerateAuxiliaryData/Scripts/Search.GenerateAuxiliaryData.cmd [hunk 1]
```diff
@@ -7,7 +7,16 @@ cd bin
...
+    start /w search.generateauxiliarydata.exe ^
+	-VaultName "#{Deployment.Azure.KeyVault.VaultName}" ^
+	-ClientId "#{Deployment.Azure.KeyVault.ClientId}" ^
+	-CertificateThumbprint "#{Deployment.Azure.KeyVault.CertificateThumbprint}" ^
+	-PrimaryDestination #{Jobs.search.generateauxiliarydata.Storage.Primary} ^
+	-PackageDatabase "#{Jobs.search.generateauxiliarydata.PackageDatabase}" ^
+	-StatisticsDatabase "#{Jobs.search.generateauxiliarydata.StatisticsDatabase}" ^
+	-AzureCdnCloudStorageAccount "#{Jobs.stats.createazurecdnwarehousereports.AzureCdn.CloudStorageAccount}"
+	-AzureCdnCloudStorageContainerName "#{Jobs.stats.createazurecdnwarehousereports.AzureCdn.CloudStorageContainerName}"
+	-verbose true -Sleep #{Jobs.search.generateauxiliarydata.Sleep} -InstrumentationKey "#{Jobs.search.generateauxiliarydata.ApplicationInsightsInstrumentationKey}"
```
Comment (joelverhagen) [inline src/Search.GenerateAuxiliaryData/Scripts/Search.GenerateAuxiliaryData.cmd]: nit: newline between these 3 params too

File: src/Search.GenerateAuxiliaryData/Exporter.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Search.GenerateAuxiliaryData
+{
+    // Public only to facilitate testing.
+    public abstract class Exporter
+    {
+        protected ILogger<Exporter> _logger;
+
+        public CloudBlobContainer DestinationContainer { get; }
```
Comment (joelverhagen) [inline src/Search.GenerateAuxiliaryData/Exporter.cs]: Why is this a property instead of a `protected` field? Seems like an implementation detail.

**Comment by joelverhagen** [comment]
Please assess feasibility of using a storage abstraction to allow for unit testing. Also, gallery storage abstraction already has a copy method that does the polling. This is now duplicated code.

**Comment by skofman1** [comment]
@joelverhagen , I know this is duplicate code and uses no storage abstraction. However, rewriting this job is out of scope for this task.

**Pull request merged**